ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,457 @@
+            }
+        }
+    }
+
+    /// Set `titleNumberOfLines` in order to control how many lines the title has. Setting this property will refresh the layout constrinats to adjust to the change
+    @objc open var twoLineTitle: Bool = false {
+        didSet {
+            if twoLineTitle != oldValue {
+                primaryLabel.numberOfLines = Constants.twoLineTitle
+                setupLayoutConstraints()
+            }
+        }
+    }
+
+    /// Set `customBackgroundColor` in order to set the background color when using the custom color style
+    @objc open var customBackgroundColor: UIColor = Constants.defaultBackgroundColor {
+        didSet {
+            if customBackgroundColor != oldValue {
+                setupColors()
+            }
+        }
+    }
+
+    /// Set `customTitleColor` in order to set the title's text color when using the custom color style
+    @objc open var customTitleColor: UIColor = Constants.defaultTitleColor {
+        didSet {
+            if customTitleColor != oldValue {
+                setupColors()
+            }
+        }
+    }
+
+    /// Set `customSubtitleColor` in order to set the subtitle's text color when using the custom color style
+    @objc open var customSubtitleColor: UIColor = Constants.defaultSubtitleColor {
+        didSet {
+            if customSubtitleColor != oldValue {
+                setupColors()
+            }
+        }
+    }
+
+    /// Set `customIconTintColor` in order to set the icon's tint color when using the custom color style
+    @objc open var customIconTintColor: UIColor = Constants.defaultIconTintColor {
+        didSet {
+            if customIconTintColor != oldValue {
+                setupColors()
+            }
+        }
+    }
+
+    /// Set `customBorderColor` in order to set the border's color when using the custom color style
+    @objc open var customBorderColor: UIColor = Constants.defaultBorderColor {
+        didSet {
+            if customBorderColor != oldValue {
+                setupColors()
+            }
+        }
+    }
+
+    /// Set `customWidth` in order to set the width of the card
+    @objc open var customWidth: CGFloat {
+        didSet {
+            if customWidth != oldValue {
+                setupLayoutConstraints()
+            }
+        }
+    }
+
+    /// The style of the card; vertical or horizontal. Style determines the layout of the content
+    private var style: CardStyle = .horizontal
+
+    /// The view of the icon. Appears on top of the text in vertical cards and besides the text in horizontal cards
+    private var iconView: UIImageView
+
+    /// The label for the Card's title
+    private let primaryLabel: Label = {
+        let primaryLabel = Label()
+        primaryLabel.translatesAutoresizingMaskIntoConstraints = false
+        primaryLabel.adjustsFontForContentSizeCategory = true
+        primaryLabel.font = TextStyle.subhead.font
+        primaryLabel.textColor = Colors.textPrimary
```

Comment:
you just need to do primaryLabel.colorstyle is by default regular so you don't need to set the color